plume-front/src/editor.rs

```diff
@@ -100,6 +216,11 @@ fn filter_paste(elt: &HtmlElement) {
 }
 
 pub fn init() -> Result<(), EditorError> {
+    if let Some(ed) = document().get_element_by_id("plume-fallback-editor") {
+        window().set_timeout(autosave, 15000);
```

Comment:
I guess set_interval does not exist? :confused:

Reply:
It doesn't look like it, oddly.

Reply:
Okay, works fine in the `js!`